sections/04_study.md

```diff
@@ -478,114 +478,103 @@ within neural networks.
-architecture formed by one 1D residual neural network and one 2D residual neural network.
-Blindly tested in the latest CASP competition (i.e. CASP12 [@url:http://www.predictioncenter.org/casp12/rrc_avrg_results.cgi]),
-RaptorX-Contact is ranked first in terms of the total F1 score (a widely-used performance metric) on
-free-modeling targets as well as the whole set of targets. In the CASP12 test, the group ranked second
-also employed a deep learning method. Even MetaPSICOV, which ranked third in CASP12, employed more
-and wider hidden layers than its old version. Wang and Xu et al. have also
-demonstrated in another blind test CAMEO (which can be interpreted as a fully-automated
-CASP) [@url:http://www.cameo3d.org/] that their predicted contacts can help fold quite a few proteins
-with a novel fold and only 65-330 sequence homologs and that their method also works well on membrane
-protein contact prediction even if trained mostly by non-membrane proteins. In fact, most of the top 10
-contact prediction groups in CASP12 employed some kind of deep learning techniques. The RaptorX-Contact
-method performed better mainly due to introduction of residual neural networks and exploiting contact
-occurrence patterns by simultaneous prediction of all the contacts in a single protein.
-It is still possible to further improve contact prediction by studying new deep network architectures.
-However, current methods fail when proteins in question have almost no sequence homologs. It is unclear if there
-is an effective way to deal with this type of proteins or not except waiting for more sequence homologs.
-Finally, the deep learning methods summarized above also apply to interfacial contact prediction
-of a protein complex, but may be less effective since on average protein complexes have fewer sequence homologs.
+beta strand and loop regions) or eight finer-grained states. A predictor is
+typically evaluated by 3-state (i.e. Q3) and 8-state (i.e. Q8) accuracy,
+respectively. Initially, several groups [@doi:10.1371/journal.pone.0032235
+@doi:10.1109/TCBB.2014.2343960 @doi:10.1038/srep11476] made important technical
+advances in adapting deep learning to protein structure prediction, but were
+unable to achieve significant improvement over the industry standard PSIPRED
+[@doi:10.1006/jmbi.1999.3091], which uses two shallow feedforward neural
+networks. In 2014, Zhou and Troyanskaya demonstrated that they could improve Q8
+accuracy by using a deep supervised and convolutional generative stochastic
+network [@arxiv:1403.1347]. In 2016 Wang et al. developed a deep CNF
+model (DeepCNF) that significantly improved Q3 and Q8 accuracy as well as
+prediction of solvent accessibility and disorder regions [@doi:10.1038/srep18962
+@doi:10.1007/978-3-319-46227-1_1]. DeepCNF was the first tool to achieve Q3
+accuracy of 84-85%, much higher than the 80% accuracy maintained by PSIPRED for
```

Comment:
Should we add a word like "80% accuracy standard" rephrase the latter part of this sentence?